setup.py

```diff
@@ -24,7 +24,7 @@
     ''',
     package_data = { '': ['README.rst'] },
     tests_require=tests_require,
-    install_requires=['rq>=0.3.5'] + tests_require,
+    install_requires=['rq>=0.3.5','croniter==0.3.5'] + tests_require,
```

Comment:
Should this be `croniter>=0.3.5`? Pinning a package dependency to a specific version is a bad idea, I think.